src/components/Card.js

```diff
@@ -24,12 +34,21 @@ export class Card {
     //We can shorten out the cardImageEl by making a function within this class and passing it through for simplicity sakes, but focus on making the rough end of the code.
   }
 
+  updateLikes(isLiked) {
+    this._isLiked = isLiked;
+    this._handleLikeIcon();
+  }
+
   _handleLikeIcon() {
     const likeButton = this._element.querySelector(".card__like-button");
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.